src/components/home/Landing.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Link from "next/link";
+import { Button } from "@/components/ui/button";
+
+const Landing = () => {
+  return (
+    <div className="bg-aisc-gray flex h-[70vh] w-screen flex-col items-center justify-center">
+      <div className="font-aisc-main gap-0 pb-12 text-center text-6xl text-white">
```

Comment:
```suggestion
    <div className="bg-aisc-gray flex h-[70vh] w-full flex-col items-center justify-center">
      <div className="font-aisc-main pb-12 text-center text-6xl text-white">
```
gap-0 is set by default and w-full is safer for overflow